pluto_build/sql/_create.sql

```diff
@@ -8,7 +8,7 @@ CREATE TABLE pluto_input_research (
     reason text,
     version text
 );
-\COPY pluto_input_research FROM 'data/pluto_input_research.csv' DELIMITER ',' CSV HEADER;
+\COPY pluto_corrections FROM 'data/pluto_input_research.csv' DELIMITER ',' CSV HEADER;
```

Comment:
not necessarily needs to be done now - just thinking that it might be good to stay consistent with what we mean by `pluto_corrections` if we are going to just use one main corrections file (`pluto_corrections + pluto_input_research`)

Reply:
*as a file that yall edit, to avoid any confusion